src/Drivers/check_determinant.cpp

```diff
@@ -0,0 +1,217 @@
+////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source
+// License.  See LICENSE file in top directory for details.
+//
+// Copyright (c) 2016 Jeongnim Kim and QMCPACK developers.
+//
+// File developed by: M. Graham Lopez, Oak Ridge National Lab
+//
```

Comment:
Jeongnim developed the original one. It is better to add her name here.

Reply:
Okay, I don't mind either way. I was trying to spare her potential emails about some code she may not want to support :)